pkg/plugin/state.go

```diff
@@ -585,7 +585,18 @@ func (e *AutoscaleEnforcer) handleStarted(logger *zap.Logger, pod *corev1.Pod) {
 
 	logger.Info("Handling Pod start event")
 
-	_, _, _ = e.reserveResources(context.TODO(), logger, pod, "Pod started", false)
+	_, _, _ = e.reserveResources(context.TODO(), logger, pod, "Pod started", reserveOptions{
+		// pod already started, out of our control - we don't have a mechanism to deny it
+		allowDeny: false,
+		// this may be a preexisting VM. If so, we should include it in "buffer" as long it's
+		// supposed to be handled by us (otherwise, the "buffer" will never be resolved)
+		includeBuffer: pod.Spec.SchedulerName == e.state.conf.SchedulerName,
```

Comment:
Why do we even have SchedulerName as a part of the spec, why do we ever want it to be different?

Reply:
Result of discussing this: it's part of the pod spec because neonvm is a logically separate dependency of "autoscaling", so the ability to run with/without the rest of autoscaling requires the ability to set the pod's scheduler name